forest/daemon/src/main.rs

```diff
@@ -92,7 +91,8 @@ fn build_daemon<'a>(config: &DaemonConfig) -> anyhow::Result<Daemon<'a>> {
     Ok(daemon)
 }
 
-fn main() {
+#[tokio::main]
+async fn main() {
```

Comment:
I'm not sure this is correct. Booting the tokio runtime _before_ we fork and daemonize Forest is tricky. If you are 100% sure it is correct, write a long-form comment explaining the reasons.

Reply:
Ah is it, I'm not sure of the issues with this, so i've moved it within with a `Runtime::block_on`